chrono-tz/Cargo.toml

```diff
@@ -30,7 +31,7 @@ include = [
 [dependencies]
 arbitrary = { version = "1.2", optional = true, features = ["derive"] }
 chrono = { version = "0.4.24", default-features = false }
-serde = { version = "1", optional = true, default-features = false }
+serde = { version = "1.0.99", optional = true, default-features = false }
```

Comment:
Did you actually run `minimal-versions` for this? How did you arrive at this version?